pkg/server/server_http.go

```diff
@@ -17,8 +17,10 @@ import (
 
 	"github.com/NYTimes/gziphandler"
 	"github.com/cockroachdb/cmux"
+	"github.com/cockroachdb/cockroach/pkg/ccl/streamingccl"
```

Comment:
oh this is no bueno, i'll have to inject this value or move the cluster setting to a non-ccl location.